MDC-104/ObjectiveC/Starter/Shrine/Shrine/ApplicationScheme.h

```diff
@@ -30,6 +31,7 @@
 - (instancetype)init NS_UNAVAILABLE;
 
 @property(nonnull, readonly, nonatomic) MDCSemanticColorScheme *colorScheme;
+@property(nonnull, readonly, nonatomic) MDCShapeScheme *shapeScheme;
```

Comment:
If this is in the 104 starter, shouldn't 103 Complete also be changed in this PR?

Reply:
103 doesn't consist of any preliminary shape stuff, and that is only added in the starter of 104.

Reply:
I see, so it just sort of appears "by magic" between the codelabs.